docs/content/docs/build/data-modeling.md

```diff
@@ -105,6 +104,8 @@ map.on("valueChanged", (changed) => {
 }
 ```
 
+For more information about handles see [Handles]({{< relref "handles.md" >}}).
```

Comment:
The next file has the "see also" as a note callout.  Should we standardize on the same format for all these "see also" bits? From the reader POV idk if it makes much difference but they do show up a lot

Reply:
I like the idea. Filed #7221 to track it separately.